public/systems_monitoring_graphs.js

```diff
@@ -0,0 +1,578 @@
+                .attr("y", 0 - Margin.left)
+                .attr("x", 0 - Height / 2)
+                .attr("dy", "1em")
+                .style("text-anchor", "middle")
+                .text("Disc Usage (GB)");
+                
+            let yLimit = data[0].disk_total
+            // Add Y axis
+            let decimalFormatter = d3.format(".2s");
+            let y = d3.scaleLinear()
+                // Add 100 to yLimit to increase brushing area
+                .domain([0, yLimit + 100])
+                .range([ Height, 0 ]);
+                svg.append("g")
+                .call(d3.axisLeft(y).ticks(5).tickFormat((d) => decimalFormatter(d).replace('G', 'GB')))      
+
+            // Add the Y gridlines
+            svg.append("g")			
+                .attr("class", "diskGrid")
+                .call(d3.axisLeft(y)
+                    .tickSize(-Width)
+                    .tickFormat("")
+                )
+
+            // Add a clipPath: everything out of this area won't be drawn.
+            let clip = svg.append("defs").append("svg:clipPath")
+                .attr("id", "clip")
+                .append("svg:rect")
+                .attr("width", Width )
+                .attr("height", Height )
+                .attr("x", 1)
+                .attr("y", 0);
+
+            // Add brushing
+            let brush = d3.brushX()                 // Add the brush feature using the d3.brush function
+                .extent( [ [0,0], [Width, Height] ] ) // initialise the brush area: start at 0,0 and finishes at width,height: it means I select the whole graph area
+                .on("end", updateChart) // Each time the brush selection changes, trigger the 'updateChart' function
+
+            // Create the scatter variable: where both the circles and the brush take place
+            let areaChart = svg.append('g')
+                .attr("clip-path", "url(#clip)")
+
+            // Area generator
+            let area = d3.area().x(d => x(d.datetime)).y0(y(0)).y1(d => y(d.disk_usage.used))
+
+            // Add the brushing
+            areaChart
+                .append("g")
+                .attr("class", "brush")
+                .call(brush);
+
+            // Add the area
+            areaChart.append("path")
+                .datum(data)
+                .attr("class", "memoryArea")  // I add the class memoryArea to be able to modify it later on.
+                .attr("fill", "#0E86D4")
+                .attr("fill-opacity", .6)
+                .attr("stroke", "#0E86D4")
+                .attr("stroke-width", 0.2)
+                .attr("d", area)
+                .on("mouseover", function() {
+                    tooltip.style("display", null);
+                })
+                .on("mouseout", function() {
+                    tooltip.style("display", "none");
+                })
+                .on("mousemove", function(d) {
+                    var xPosition = d3.mouse(this)[0] - 15;//x position of tooltip
```

Comment:
```suggestion
                    var xPosition = d3.mouse(this)[0] - 15; // x position of tooltip
```
Code is slightly cleaner/easier to read if you put spaces around all of your '//'s. You may be able to configure your editor to warn about this automatically.

Reply:
Updated thanks.